source/Hardpoint.cpp

```diff
@@ -218,6 +218,7 @@ void Hardpoint::Fire(Ship &ship, vector<Projectile> &projectiles, vector<Visual>
 	// offset of (.5 * velocity) and that velocity includes the velocity of the
 	// ship that fired them.
```

Comment:
This comment isn't accurate anymore.

Reply:
Well projectiles still are using an offset of half the velocity as the comment mentions. It's the effects that aren't using the offset anymore, but the comment doesn't mention them. (Although we could probably update this comment to describe *why* this offset is needed to "Get projectiles to start at the right position.")

Reply:
The `0.5 * velocity` offset is explained more in BatchDrawList:
https://github.com/endless-sky/endless-sky/blob/f5f638c7c4a94b2b37d4c2006e0c50d4e8666c7f/source/BatchDrawList.cpp#L62-L66